app/helpers/jsonld_helper.rb

```diff
@@ -226,6 +226,6 @@ def load_jsonld_context(url, _options = {}, &_block)
 
     doc = JSON::LD::API::RemoteDocument.new(json, documentUrl: url)
 
-    block_given? ? yield(doc) : doc
+    _block ? yield(doc) : doc
```

Comment:
Variables with `_` as prefix should be changed to `block` instead of `_block` because it implies a variable that is not used.

Reply:
It's passed in as `&_block` so that's what the autofix kept it as

Reply:
Yes, you should change `&_block` to `&block`.